docs/embedding/tenants.md

```diff
@@ -0,0 +1,387 @@
+
+[SCREENSHOT]
+
+## Subscription permissions for tenants
+
+By default, all tenant users will be created with **No** [subscription permissions](LINK). If you want your users to be able to create subscriptions (either in full-app embedding, modular embedding, or by logging in directly to Metabase), you'll need to change the Subscription and alerts permissions to "Yes".
+
+## Deactivate a tenant
+
+**Deactivating a tenant will also deactivate all users who belong to this tenant**.
+
+To deactivate a tenant:
+
+1. Go to **Admin settings > People**.
+2. Select **Tenants** on the left sidebar.
+3. Click on **three dots** next to the tenant.
+4. Choose **Deactivate tenant**
+
+All tenant users will be deactivated and won't be able to sign in anymore. Tenant users will not be permanently deleted (Metabase does not delete users, only deactivates), so even though the tenant users will be deactivated, you won't be able to create new users with the same email.
+
+## Changing tenant strategy
+
+### From single-tenant to multi-tenant
+
+When you enable multi-tenant strategy, all users that currently exist in Metabase will be considered "internal" users. If you don't want any of those users to become [tenant users](tenant), you can just proceed with tenant setup as if you had a fresh new instance (create tenants, create collections, set up permissions, etc).
+
+However, if you want to assign some existing users to tenants, you'll need to:
+
+1. Mark them as tenant users using the API call:
+
+   ```
+   PUT /api/user/:id
+   {"tenant_id": 1}
+   ```
+
+2. If you're using JWT for SSO, [add a `@tenant` claim to your JWT](LINK).
+3. Set up [tenant groups][LINK], [data permissions][LINK], and [collection permissions](LINK) because you won't be able to use existing internal groups for tenant permissions.
+
+### From multi-tenant to single-tenant
+
+If you disable multi-tenant strategy, _all your tenant users will be deactivated_ and _all collections will be deleted_ (although you'll get both users and collections back if you reactivate multi-tenant strategy later). So if you want to wish to keep the active users but just disable tenancy features, you'll need some extra prep.
+
+1. Replicate the tenant setup you have with regular groups and collections (instead of tenant groups and shared collections). Review documentation for Data permissions, Collection permissions, and User groups. Make sure to thoroughly test your setup with test users. A [development instance](LINK) might come in handy.
+2. If you're using any tenant groups, remove the tenant group memberships of all tenant users.
+3. Change the tenant users to internal users using API:
+
+   ```
+   PUT /api/user/:id
+   {"tenant_id": null}
+   ```
+
+4. Finally, disable the feature once everything is verified to work
+
+If you don't do step 3, all your users will be deactivated.
+
+## End user experience
```

Comment:
I like this. Maybe we should move it up? As part of the intro/why-would-you-even-set-up-tenants section?